lib/convenient_service/service/plugins/has_result_status_check_short_syntax/concern.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module ConvenientService
+  module Service
+    module Plugins
+      module HasResultStatusCheckShortSyntax
+        module Concern
+          include Support::Concern
+
+          class_methods do
+            def success?(...)
+              result(...).success?
+            end
+
+            def error?(...)
+              result(...).error?
+            end
+
+            def failure?(...)
+              result(...).failure?
+            end
+
+            def not_success?(...)
+              result(...).not_success?
+            end
+
+            def not_error?(...)
+              result(...).not_error?
+            end
+
+            def not_failure?(...)
+              result(...).not_failure?
+            end
```

Comment:
@Olha-30 

Could you add YARD method signature docs, please?

For example:

```ruby 
##
# @return [Boolean]
# @since 0.2.0
#
def success?(...)
  result(...).success?
end
```

Thanks.